### DISCUSSION TITLE
Comparison with Runno

### DISCUSSION THREAD

**Discussion by aehlke** [live-codes/livecodes#424] [General]
I've just come across a similar looking service, https://runno.dev

Thought I'd share here for any inspiration or open source reuse potential, since wasm/browser-local code sandboxes like LiveCodes are still so rare

**Comment by hatemhosny** [comment]
Thank you @aehlke 
A very nice project indeed.
I will have a deeper look.

**Comment by dwina** [comment]
the resource loader in livecodes not work. I am not able to run material ui in the version version26 selfhosting (npm install livecodes | npm install @material-ui/core)

Any suggestion?

**Reply by hatemhosny** [reply]
Thank you @dwina for reaching out.
Can you please open a [new issue](https://github.com/live-codes/livecodes/issues) and provide a [reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) so that I can try to help?
Thank you for understanding.
